main/src/ca/uwaterloo/flix/language/phase/Safety.scala

```diff
@@ -61,15 +65,15 @@ object Safety {
     * @param inTryCatch indicates whether `exp` is enclosed in a try-catch. This can be reset if the
     *                   expression will later be extracted to its own function (e.g [[Expr.Lambda]]).
     */
-  private def visitExp(exp0: Expr)(implicit inTryCatch: Boolean, renv: RigidityEnv, flix: Flix): List[SafetyError] = exp0 match {
+  private def visitExp(exp0: Expr)(implicit inTryCatch: Boolean, renv: RigidityEnv, flix: Flix, sctx: SharedContext): Unit = exp0 match {
```

Comment:
Here I would consider inTryCatch every specific. renv pretty specific, sctx specific, and flix non-specific, so I would request the order:

implicit inTryCatch: Boolean, renv: RigidityEnv, sctx: SharedContext, flix: Flix,